autorag/nodes/queryexpansion/hyde.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+from typing import List
+
+from llama_index.llms.llm import BaseLLM
+
+hyde_prompt = "Please write a passage to answer the question"
+
+
+def hyde(queries: List[str], llm: BaseLLM,
+         prompt: str = hyde_prompt) -> List[List[str]]:
```

Comment:
get llm extra parameters as dict in here, so it can adjust llm setting by users.

Reply:
l'm going to resolve it query_expansion decorator.
coming soon